### DISCUSSION TITLE
Passive BLE monitor non funziona dopo aggiornamento python 3.9

### DISCUSSION THREAD

**Discussion by mkel84** [indomus/forum#1276] [HUB: Home Assistant]
Salve a tutti,
dopo aver aggiornato l'venv di home assistant alla versione 3.9 di python l'integrazione Passive BLE monitor non va più. il log di home assistant mi da questo errore:

Questo errore ha avuto origine da un'integrazione personalizzata.

Logger: custom_components.ble_monitor
Source: custom_components/ble_monitor/__init__.py:654
Integration: Passive BLE monitor (documentation, issues)
First occurred: 19:32:37 (32 occurrences)
Last logged: 21:05:49

HCIdump thread: Runtime error while stop scan request on hci0: Event loop stopped before Future completed..

Ho provato a cercare sulla rete ma non sono riuscito a trovare una soluzione, qualcuno sa come risolvere il problema?

**Comment by Scialla** [comment]
Successo anche a me.... rimuovi completamente l'integrazione, riavvia HA, reinstalla l'integrazione

**Reply by mkel84** [reply]
Ho rimosso l'integrazione da HA, ho cancellato fisicamente la cartella da custom_components, riscaricato l'integrazione e continua a darmi quell'errore, ho sbagliato qualcosa?

**Reply by Scialla** [reply]
Io non ho cancellato la cartella, ma ho riavviato HA e dopo ho reinstallato l'integrazione

**Reply by mkel84** [reply]
Ho provato a fare anche io la stessa cosa ma continua a darmi lo stesso errore

**Reply by Scialla** [reply]
prova a dare questo comando
`sudo setcap cap_net_raw+eip $(eval readlink -f `which node`)`

Se non va segui la issue:
https://github.com/custom-components/ble_monitor/issues/295
